cpp/src/strings/convert/convert_booleans.cu

```diff
@@ -90,6 +90,32 @@ std::unique_ptr<column> to_booleans(strings_column_view const& strings,
 }
 
 namespace detail {
+
+namespace {
+struct from_booleans_fn {
+  column_device_view const d_column;
+  string_view d_true;
+  string_view d_false;
+  offset_type* d_offsets{};
+  char* d_chars{};
+
+  __device__ void operator()(size_type idx) const
+  {
+    if (d_column.is_null(idx)) {
+      if (d_chars == nullptr) d_offsets[idx] = 0;
+      return;
+    }
+
+    if (d_chars != nullptr) {
+      auto const result = d_column.element<bool>(idx) ? d_true : d_false;
```

Comment:
This doesn't need template keyword after the dot operator to compile?

Reply:
No. You generally only need that if the function this is called in is also templated in order to help the compiler resolve the typenames correctly.